CONTRIBUTING.adoc

```diff
@@ -0,0 +1,25 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/code-ready/crc/issues[GitHub issue tracker] for the repository.
```

Comment:
Is there a reason for the new line between "...the standard" and the URL here?

Reply:
Also, for docs, we try to use explicit links when possible using `link:<url>[<name>]` rather than relying on the URL being automatically detected. :)